### PR TITLE
Update

### PR DESCRIPTION
**Pull request by bbhtt**
The list of dependencies was getting too large, this can be easily generated with flatpak-pip-generator.

I'm open to dropping the home <- host change if necessary :) (in my opinion home should be enough and users can always add the directories they need access to using `flatpak override`)

This built and ran fine locally.

Ping @cvfosammmm, please review when you have time, thanks!

### CODE REVIEW

**Comment by bbhtt** [comment]
bot, build

**Comment by bbhtt** [comment]
> Started [test build 101314](https://buildbot.flathub.org/#/builders/32/builds/101314)

This build will probably fail because I forgot to add qhull as a dep of matplotlib

**Comment by bbhtt** [comment]
bot, build

**Comment by bbhtt** [comment]
bot, build

**Comment by bbhtt** [comment]
Hopefully this one works. The issue is pip generator doesn't account for setup dependencies and neither are they mentioned [here](https://matplotlib.org/stable/devel/dependencies.html).

**Comment by bbhtt** [comment]
bot, build

**Comment by bbhtt** [comment]
> Started [test build 101428](https://buildbot.flathub.org/#/builders/32/builds/101428)

```
  WARNING: Generating metadata for package kiwisolver produced metadata for project name unknown. Fix your #egg=kiwisolver fragments.
Discarding file:///run/build/python3-matplotlib/kiwisolver-1.4.4.tar.gz: Requested unknown from file:///run/build/python3-matplotlib/kiwisolver-1.4.4.tar.gz (from matplotlib) has inconsistent name: filename has 'kiwisolver', but metadata has 'unknown'
ERROR: Could not find a version that satisfies the requirement kiwisolver>=1.0.1 (from matplotlib) (from versions: 1.4.4)
ERROR: No matching distribution found for kiwisolver>=1.0.1
```

This is weird... it has the correct project name.

File: org.cvfosammmm.Porto.json [hunk 1]
```diff
@@ -1,376 +1,73 @@
...
         },
         {
             "name": "porto",
             "buildsystem": "simple",
             "build-commands": [
-                "source /app/bin/activate",
-                "mkdir -p /app/share/applications",
-                "cp resources/org.cvfosammmm.Porto.desktop /app/share/applications/org.cvfosammmm.Porto.desktop",
+                "mkdir -p ${FLATPAK_DEST}/share/applications",
+                "cp resources/org.cvfosammmm.Porto.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
```
Comment (hfiguiere) [inline org.cvfosammmm.Porto.json]:
```suggestion
                "install -Dm644 resources/org.cvfosammmm.Porto.desktop  ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop",
```

Comment (hfiguiere) [reply org.cvfosammmm.Porto.json]: annd similar below but I can't suggest because github.

tl;dr install does create the directories and is better than `cp` to install files.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.cvfosammmm.Porto.json (hunk 1 above)
Comment (bbhtt) [reply org.cvfosammmm.Porto.json]: Thanks!

**Comment by bbhtt** [comment]
bot, build
